worker/snapshot_test.go

```diff
@@ -34,6 +34,7 @@ import (
 )
 
 func TestSnapshot(t *testing.T) {
+	t.Skip()
```

Comment:
IMO it's good practice to supply an argument to Skip that indicates why the test is being skipped.

Reply:
Removed it